src/requestCertificate.jsx

```diff
@@ -0,0 +1,354 @@
+                <Radio isChecked={dialogValues.storage === "nssdb"}
+                    name="nssdb"
+                    onChange={() => onValueChanged("storage", "nssdb")}
+                    label="NSSDB"
+                    id="nssdb"
+                    value="nssdb" />
+                <Radio isChecked={dialogValues.storage === "file"}
+                    name="file"
+                    onChange={() => onValueChanged("storage", "file")}
+                    label="File"
+                    id="file"
+                    value="file" />
+            </div>
+        </>
+    );
+}
+
+const CAsRow = ({ onValueChanged, dialogValues, cas }) => {
+    return (
+        <>
+            <label className="control-label" htmlFor="ca">
+                {_("CA")}
+            </label>
+            <Select.Select id="ca"
+                initial={dialogValues.ca}
+                onChange={value => onValueChanged("ca", value)}>
+                    {cas.map(ca => {
+                        return (
+                        <Select.SelectEntry data={ca.nickname.v} key={ca.nickname.v}>
+                            {ca.nickname.v}
+                        </Select.SelectEntry>
+                        );
+                    })}
+            </Select.Select>
+        </>
+    );
+}
+
+const SetSigningParametersRow = ({ onValueChanged, dialogValues }) => {
+    return (
+        <label className='checkbox-inline'>
+            <input id='set-signing-parameters'
+                type='checkbox'
+                checked={dialogValues.signingParameters}
+                onChange={e => onValueChanged("signingParameters", !dialogValues.signingParameters)} />
+            {_("Set Signing Request Parameters")}
+        </label>
+    );
+}
+
+const SubjectNameRow = ({ onValueChanged, dialogValues }) => {
+    return (
+        <>
+            <label className="control-label" htmlFor="subject-name">
+                {_("Subject Name")}
+            </label>
+            <TextInput value={dialogValues.subjectName}
+                id="subject-name"
+                type="text"
+                onChange={(value) => onValueChanged("subjectName", value)}
+                aria-label={_("SubjectName input text")} />
+        </>
+    );
+}
+
+const DNSNameRow = ({ onValueChanged, dialogValues }) => {
+    return (
+        <>
+            <label className="control-label" htmlFor="dns-name">
+                {_("DNS Name")}
+            </label>
+            <TextInput value={dialogValues.dnsName}
+                id="dns-name"
+                type="text"
+                onChange={(value) => onValueChanged("dnsName", value)}
+                aria-label={_("DNSName input text")} />
+        </>
+    );
+}
+
+const PrincipalNameRow = ({ onValueChanged, dialogValues }) => {
+    return (
+        <>
+            <label className="control-label" htmlFor="principal-name">
+                {_("Principal Name")}
```

Comment:
Should be Sentence style rather than Headline Style

Reply:
Fixed this and all above